app/views/validators/_spark_line_skipped_vote.html.erb

```diff
@@ -5,9 +5,9 @@
         type: 'gauge',
         data: {
             datasets: [{
-                data: [-5, -3.0, 0.05],
+                data: [<%= -(@batch.skipped_vote_all_median * 20) %>, <%= -(@batch.skipped_vote_all_median * 10) %>, 0.05],
```

Comment:
Why we multiplicate by 10 and 20 and by 30 below? Maybe you could create some meaningful method in the model to get rid of calculations in the view?

Reply:
I don't think it should be placed in the model - it does not represent any data other than ranges in frontend chart.

Reply:
Data is multiplicated for chart purposes only - first range is eqal median, second median*2, and so on.

Reply:
However, please consider using some variables like - for someone who does not work on that these numbers are confusing.

Reply:
@maciejkocylapc I'd pass them in render helper like account and value, I think that will be clear enough